providers/netcore/src/lib/netcore.provider.ts

```diff
@@ -57,6 +57,20 @@ export class NetCoreProvider implements IEmailProvider {
     body.personalizations[0].to[0] = new netcoreLib.EmailStruct();
     body.personalizations[0].to[0].email = options.to;
 
+    if (options.cc) {
+      body.personalizations[0].cc = options.cc.map((ccItem, index) => {
+        body.personalizations[0].cc[index] = new netcoreLib.EmailStruct();
+        body.personalizations[0].cc[index].email = ccItem;
+      });
```

Comment:
`map` needs to return a value, here `undefined` is returned so i don't think it will work, we should either use a forEach or a for loop if we are not returning .

I would suggest a cleaner approach perhaps:

```suggestion
      body.personalizations[0].cc = options.cc.map((ccItem) => {
        const emailStruct = new netcoreLib.EmailStruct();
        emailStruct.email = ccItem;
        
        return emailStruct;
      });
```